File: C3-Assignments/C3-Assignment-03/03.cpp
//03. Write a C++ program to print first N even natural numbers.
#include<iostream>
using namespace std;
int main()
{
    int n, i;
    cout<< "Enter a number: ";
    cin>>n;
    n*=2;
    for(i=1;i<=n;i++)
    {
        if(i%2==0)
            cout<< i << " ";
    }
    return 0;
}

File: Python/PY-25 AS 11 more on while/03.py
#   03. Write a python script to calculate sum of cubes of first N natural numbers.
n = int(input("Enter a number: "));
print("Sum of cubes of first %d natural number is: "%(n),end="");
sum = 0;
while n:
    sum = sum + (n*n*n);
    n-=1;
print(sum);